server/src/main/java/org/opentosca/toscana/plugins/cloudfoundry/CloudFoundryLifecycle.java

```diff
@@ -91,31 +179,52 @@ private boolean isNotNull(String... elements) {
 
     @Override
     public void transform() {
-        Application myApp = new Application(1, context);
+        logger.info("Begin transformation to Cloud Foundry.");
```

Comment:
This sentence sounds wrong. Maybe consider Rewording to:
`Starting transformation phase...`

Mentioning that you ttransform to CludFoundry sounds irrelevant, beacause this is kind of obvious anyways.

Reply:
This could also be redundant though, because the Abstract lifecycle also produces output on which phases it currently executes